libs/core/src/components/step-list/step-list-item.ts

```diff
@@ -0,0 +1,133 @@
+import { LitElement, PropertyValues, css } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import "../icon/icons/checkmark.js";
+import { html } from "../../scoping.js";
+
+enum LineColor {
```

Comment:
Colors should come from chlorophyll or green-tokens.

This is 2016 design (the current SEB design) so the component should use chlorophyll. Here's [an example how to use the colors from chlorophyll](https://github.com/seb-oss/green/blob/main/libs/core/src/components/button/button.trans.styles.scss).

Probably a good idea to break out the styling to a SCSS file so we keep things seperate for readability.

Reply:
Hi, I am back from Vacation and sick leave :D hopefully in coming sprint, I could take out some time and fix this issue and then we can also book some pair programming session as you suggested before!